release.yaml

```diff
@@ -1,3 +1,61 @@
+fleet:
+- 100.0.3+up0.3.9
```

Comment:
additional charts should be deleted from release.yaml; these were removed in the webhook PR